test/parallel/test-util-promisify.js

```diff
@@ -184,3 +184,11 @@ const stat = promisify(fs.stat);
     })
   ]);
 }
+
+common.expectsError(
+  () => promisify(undefined),
+  {
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: Error,
```

Comment:
Please use `TypeError` here instead of `Error`.

Reply:
I am going to open a PR to make that stricter.

Reply:
@Trott : I've updated that. Kindly review now. Thanks !

Reply:
Yes, that looks 👍 to me. Thanks.